cmake/CommonDefs.cmake

```diff
@@ -254,7 +235,7 @@ macro(add_opendf_module)
         else(DEFINED BOLD_HYB_GLOBAL_BUILD)
             string(REGEX REPLACE "^opendf_" "" pkgcomp_ ${pkg_})
             find_package(opendf QUIET COMPONENTS ${pkgcomp_})
-            if (opendf_${pkgcomp_}_FOUND) 
```

Comment:
this probably should be pomerol, not opendf :)

Reply:
Um, why does this project need a macro called `add_opendf_module()`, which also refers to `BOLD_HYB_GLOBAL_BUILD`? :)
It is never called in any CMake script.

Reply:
History of copy-pasting between codes be like... :)

Reply:
Let's add a TODO to clean this if you don't want to address it now.